spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -36,22 +36,23 @@
       scan_count: response['scanCount'],
       secondary_id_type: response['secondaryIdType'],
       status: response['status'],
-      transaction_end_date_time: response['transactionEndDateTime'],
-      transaction_start_date_time: response['transactionStartDateTime'],
+      transaction_end_date_time: anything,
+      transaction_start_date_time: anything,
```

Comment:
These are verified below in specific tests since not all tests will set these values (eg expired enrollments)